cmd/routecapture.go

```diff
@@ -46,12 +47,13 @@ func (a *WebScan) InitRoutecaptureCommand() {
 			}
 
 			baseURLsOnly, _ := cmd.Flags().GetBool("base-urls-only")
+			captureStaticAssets, _ := cmd.Flags().GetBool("capture-static-assets")
 
 			timeout, _ := cmd.Flags().GetInt("timeout")
 			minDOMStabalizeTime, _ := cmd.Flags().GetInt("minDOMStabalizeTime")
 
 			// Extract the routes and links
-			report := routecapture.PerformRouteCapture(cmd.Context(), target, webscan.PageCaptureMethodRequest, baseURLsOnly, timeout, minDOMStabalizeTime, insecure, browserPath, nil, nil, nil)
+			report := routecapture.PerformRouteCapture(cmd.Context(), target, webscan.PageCaptureMethodRequest, baseURLsOnly, captureStaticAssets, timeout, minDOMStabalizeTime, insecure, browserPath, nil, nil, nil)
```

Comment:
At some point soon I'd suggest we create an Opts struct that we can pass in vs the ever growing list of arguments.